Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM woorank/docker-node-babel
+EXPOSE 8080-8082
+
+ADD node_modules/ /node_modules
```

Comment:
Hm, I am not sure if this doesn't introduce a conflict between host OS and container OS in case these modules contain pre-compiled native code